graphics/Pangolin/Portfile

```diff
@@ -19,9 +19,21 @@ long_description    Pangolin is a lightweight portable rapid development library
                     Pangolin also provides a mechanism for manipulating program variables through config files \
                     and ui integration, and has a flexible real-time plotter for visualising graphical data.
 
-depends_lib-append  port:glew
+depends_build-append \
+                    port:pkgconfig
+
+depends_lib-append  port:glew \
+                    port:openexr \
+                    port:libdc1394 \
+                    port:zlib \
+                    port:libpng \
+                    port:jpeg \
+                    port:tiff \
+                    path:lib/libavcodec.dylib:ffmpeg
```

Comment:
Please use the incantation we normally use in other ports:
`path:bin/ffmpeg:ffmpeg`

Reply:
Ports that use the ffmpeg binary should use `path:bin/ffmpeg:ffmpeg`. (There are currently 3 ports doing so.) This port uses the ffmpeg libraries, so it should use `path:lib/libavcodec.dylib:ffmpeg`.  (There are currently 103 ports doing so.)

Reply:
But pangolin is not associated with ffmeg application, it only requires libraries. For example opencv use path:lib/libavcodec.dylib:ffmpeg

Reply:
Well whatever Ryan thinks is best is good with me! :)